metis-dereference/metis-dereference-service/src/main/java/eu/europeana/metis/dereference/service/utils/RdfRetriever.java

```diff
@@ -96,6 +96,8 @@ private static String retrieveFromSource(String resourceId, String url, int redi
       throws IOException {
 
     // Make the connection and retrieve the result.
+    // Note: we have no choice but to follow the provided URL.
+    @SuppressWarnings("findsecbugs:URLCONNECTION_SSRF_FD")
```

Comment:
Normally we were marking the issue in sonarqube so that it's not considered in the analysis.
I don't know if applying that annotation would work, but we can give it a try otherwise if it doesn't work we might want to remove it. 
Same for the other annotations

Reply:
It does work, I tested it. I used it for Clio as well (and it also worked there).
This way, there is less chance of it being lost when the code is changed.
And it even generates a (much lower priority) warning in SonarQube that ensures that we don't completely forget about it.

Reply:
Cool okay let's see how this pans out.